deploy-edgeX.sh

```diff
@@ -77,7 +77,7 @@ run_service export-distro
 
 run_service app-service-rules
 
-run_service rulesengine
+[ "$(uname -m)" = "x86_64" ] && run_service rulesengine
```

Comment:
please also remove line 94
sleep 100s # Wait for rulesengine fully startup, because it takes around 100s on Raspberry Pi

Reply:
removed